index.html

```diff
@@ -4709,6 +4708,18 @@ <h2 id="AudioBufferSourceNode">
           This node has no <a>tail-time</a> reference.
         </p>
         <p>
+          A <dfn>playhead position</dfn> for an <a>AudioBufferSourceNode</a> is
```

Comment:
I still find this odd to have this here.  As mentioned below, we don't say anything about multiplying by the sample rate and `loopStart` and `loopEnd` are just values in seconds indexing into the buffer.  I'd prefer this paragraph in the the playback section.

WDYT @padenot